docs/synthesis.md

```diff
@@ -128,18 +128,34 @@ synthesis:
 
 ### Tags
 
-You can also define a set of attributes that can be copied into entity tags. These attributes will only be copied after a datapoint matches a rule.
+You can also define a set of attributes that can be copied into entity tags.
 
+These are global tags, and they are applied for the rule or every rule from a set of rules.
```

Comment:
```suggestion
These are global tags, and they are applied for all the rules from the set of rules.
```

Reply:
Just to notice, maybe I need to rephrase it, but I would like to remark two approaches.
1. There's a unique rule, where the rule itself lives inherited and directly under "synthesis:".
2. There're multiple rules, where they live under "rules:".

Reply:
Improved, I think it's much clearer like this.